### DISCUSSION TITLE
Revalidate key for getStaticPaths

### DISCUSSION THREAD

**Discussion by samrobbins85** [vercel/next.js#20331] [Ideas]
For CMS backed applications like blogs, there will often be a dynamic path, and being able to revalidate this to check for new pages would offer a better experience for both users and developers than using the fallback key and then throwing 404 in the page if data isn't supplied

**Comment by tomdohnal** [comment]
That's what the `revalidate` property is for, isn't it?
https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation

**Reply by samrobbins85** [reply]
From my testing this will only revalidate existing pages, not add new pages
